pkg/model/iam.go

```diff
@@ -103,7 +108,7 @@ func (b *IAMModelBuilder) buildIAMTasks(igRole kops.InstanceGroupRole, iamName s
 
 		iamRole = &awstasks.IAMRole{
 			Name:      s(iamName),
-			Lifecycle: b.Lifecycle,
+			Lifecycle: &subtaskLifecycle,
```

Comment:
This should be the regular lifecycle for the security phase. Not certain why you are making a new one.